boards/shared/Src/App/App_SharedErrorTable.c

```diff
@@ -263,7 +246,13 @@ void App_SharedErrorTable_GetBoardsWithNoErrors(const struct ErrorTable *error_t
 
     for (enum Board board = 0; board < NUM_BOARDS; board++)
     {
-        if (!App_SharedError_IsBoardInList(&boards_with_errors, board))
+        // BMS not in table, always return as not containing errors
```

Comment:
I'm not sure how the error table is defined, but can we remove the BMS entirely from this table without breaking the entire system?

Reply:
To remove the BMS from the table completely we would need to remove the BMS from the Boards enum. See the discussion above.